### PR TITLE
Teltonika io32 coolant temp/io115 engine temp

### PR DESCRIPTION
**Pull request by K-J-Dod24**
Coolant temperature sent from teltonika devices comes in as the io32 but is currently written in code as io115 which is engine temperature as sent from device. This is the web side of the code to correctly define and separate the io's.

### CODE REVIEW

**Comment by tananaev** [comment]
As I already mentioned previously, you should not include any translations in the pull request. Instead the localization should be done through Transifex.

**Comment by K-J-Dod24** [comment]
This isn't for language, we have included the English so that the output displayed is written out correctly, as well as this the usePositionAttributes.js, formatter.js and positionvalue.js have been edited to correctly display °C correctly and allow for the inputs to be displayed on the popup aswell as to be charted.

**Comment by tananaev** [comment]
As I explained several times, you should never include non-English strings in your PRs. This is the first thing you have to revert.

Secondly, we only add attributes that in the `Position.java`. We can keep `coolantTemp`, but `engineTemp` has to go, unless you can add it to at least one other protocol.

File: modern/src/common/util/formatter.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ export const formatNumber = (value, precision = 1) => Number(value.toFixed(preci
 
 export const formatPercentage = (value) => `${value}%`;
 
+export const formatCelsius = (value) => `${value}°C`;
```
Comment (tananaev) [inline modern/src/common/util/formatter.js:20]: I think we should rename this to `formatTemperature`.

File: modern/src/resources/l10n/af.json [hunk 2]
```diff
@@ -231,6 +231,8 @@
     "linkGoogleMaps": "Google Maps",
     "linkAppleMaps": "Apple Maps",
     "linkStreetView": "Street View",
+    "positionEngineTemp": "Enjinsemperatuur",
```
Comment (tananaev) [inline modern/src/resources/l10n/af.json:234]: Please remove all the localization except English.

File: modern/src/resources/l10n/en.json [hunk 3]
```diff
@@ -231,6 +231,7 @@
     "linkGoogleMaps": "Google Maps",
     "linkAppleMaps": "Apple Maps",
     "linkStreetView": "Street View",
+    "positionCoolantTemp": "Coolant Temperature",
```
Comment (tananaev) [inline modern/src/resources/l10n/en.json:234]: Can you please move this down to the list, ideally in the same order it is present in the Position.java.

**Review by tananaev — COMMENTED** [review]
Looks like there's also a bunch of compilation errors.

**Comment by tananaev** [comment]
Merged, thank you.

**Pull request merged**